torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -57,41 +58,86 @@ def convert_bounding_box_format(
     return bounding_box
 
 
+def _split_alpha(image: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+    return image[..., :-1, :, :], image[..., -1:, :, :]
+
+
+def _strip_alpha(image: torch.Tensor) -> torch.Tensor:
+    image, alpha = _split_alpha(image)
+    if not torch.all(alpha == _FT._max_value(alpha.dtype)):
+        raise RuntimeError(
+            "Stripping the alpha channel if it contains values other than the max value is not supported."
+        )
+    return image
+
+
+def _add_alpha(image: torch.Tensor, alpha: Optional[torch.Tensor] = None) -> torch.Tensor:
+    if alpha is None:
+        shape = list(image.shape)
+        shape[-3] = 1
+        alpha = torch.full(shape, _FT._max_value(image.dtype), dtype=image.dtype, device=image.device)
+    return torch.cat((image, alpha), dim=-3)
```

Comment:
From what I can tell this should work for batches too. Have you confirmed?

Reply:
```py
>>> image = torch.rand(5, 4, 3, 16, 16)
>>> alpha = torch.rand(5, 4, 1, 16, 16)
>>> torch.cat((image, alpha), dim=-3)
torch.Size([5, 4, 4, 16, 16])
```